common/ai_strategy/r56_ENG.txt

```diff
@@ -1602,4 +1602,53 @@ r56_ENG_dont_turn_ECU_PRU_war_into_WW3 = {
 		id = "ECU"
 		value = -75
 	}
+}
+
+r56_ENG_dont_join_jordan = {
```

Comment:
If UK has any allies, those would probably call them in indirectly, although if only dominions, this should suffice